viewing_party/party.py

```diff
@@ -1,23 +1,165 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    
+    movie = {}
+    movie_set = {title, genre, rating}
+
+    if None in movie_set:
+        return None
+    else:

```

Comment:
Since the `if` branch returns from the function, it would be safe to remove the `else:` and unindent the remaining code, treating the `if` check as a guard clause. Guard clauses usually follow this pattern:

```python
    if some_special_case:
        return special_result

    # remaining main logic
```